VideoLocker/AndroidManifest.xml

```diff
@@ -71,7 +71,7 @@
 
     <application
         android:allowBackup="true"
-        android:supportsRtl="false"
+        android:supportsRtl="@bool/supports_rtl"
```

Comment:
The comment indicated that it did not change at all after installation in Jelly Bean though.
> On 12-Sep-2015, at 1:45 am, Brian Guertin <notifications@github.com> wrote:
> 
> In VideoLocker/AndroidManifest.xml <https://github.com/edx/edx-app-android/pull/359#discussion_r39315463>:
> 
> > @@ -71,7 +71,7 @@
> >  
> >      <application
> >          android:allowBackup="true"
> > -        android:supportsRtl="false"
> > +        android:supportsRtl="@bool/supports_rtl"
> Of course it won't change dynamically, since the manifest is only read on application start-up. If we want to support run-time language changes that's a bigger task.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/359/files#r39315463>.
>